ttnn/cpp/ttnn/operations/conv/conv2d/conv2d.cpp

```diff
@@ -740,6 +740,8 @@ std::tuple<ttnn::Tensor, uint32_t, uint32_t, ttnn::Tensor, std::optional<ttnn::T
         }
         conv_config.deallocate_activation = true;
     }
+    if(conv_config.shard_layout == TensorMemoryLayout::WIDTH_SHARDED) {  conv_config.act_block_h_override = 0; }
```

Comment:
you should fix up the usage of this arg in case of width sharding (which seems to be in determine_per_core_conv_block_config
instead of overriding it here.

Reply:
I think the best approach  would be to remove `act_block_h_override` and `act_block_w_div`, and instead have a single parameter called `act_block_div` that would divide the act_block based on the shard layout.

Reply:
agree with @pavlejosipovic . this should be part of the config determination -- and can be removed once we get rid of these parameters.
Additionally, can you update the documentation for convConfig as well to specify that this parameter is ignored for width sharding.

Reply:
Moved to `determine_per_core_conv_block_config`